doc/changing_derivatives.md

```diff
@@ -34,7 +37,12 @@ end
 *Scenario: Your app is currently working only with original files, and you want
 to introduce derivatives.*
 
-Start by loading the `derivatives` plugin and defining your processing logic:
+#### 1. Add `derivatives` plugin and `image_processing` gem
+
+Start by loading the `derivatives` plugin in the uploader and the 
+`image_processing` gem, and then define the processing logic. In this example, 
```

Comment:
The `image_processing` gem would not be needed if the user is not processing images. So it would be ideal if wording was more generic here.